tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -1961,34 +1961,41 @@ public function userOverwritesFileFromAndToSpaceBetweenSpaces(
 	}
 
 	/**
-	 * @Then /^user "([^"]*)" should not be able to download file "([^"]*)" from space "([^"]*)"$/
+	 * @Then /^user "([^"]*)" (should|should not) be able to download file "([^"]*)" from space "([^"]*)"$/
 	 *
 	 * @param string $user
+	 * @param string $shouldOrNot
 	 * @param string $fileName
 	 * @param string $spaceName
 	 *
 	 * @return void
 	 * @throws GuzzleException
 	 */
-	public function userShouldNotBeAbleToDownloadFileInsideSpace(
+	public function userShouldBeAbleToDownloadFileInsideSpace(
```

Comment:
```suggestion
	public function userShouldOrShouldNotBeAbleToDownloadFileFromSpace(
```

Reply:
I am just taking the reference from these kind of methods:
`theCLIResponseShouldContainTheseEntries`, `theCommandOutputShouldContain`, `userShouldHaveShareSharedByUserFromSpace`, etc.

Reply:
Step definition function should match step definition text.

Reply:
There are some method that were not changed when steps were modified. so please do not rely of old method names. The rule is always to match the step and method name